js/src/ui/Tags/tags.less

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2015, 2016 Ethcore (UK) Ltd.
+/* This file is part of Parity.
+/*
+/* Parity is free software: you can redistribute it and/or modify
+/* it under the terms of the GNU General Public License as published by
+/* the Free Software Foundation, either version 3 of the License, or
+/* (at your option) any later version.
+/*
+/* Parity is distributed in the hope that it will be useful,
+/* but WITHOUT ANY WARRANTY; without even the implied warranty of
+/* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/* GNU General Public License for more details.
+/*
+/* You should have received a copy of the GNU General Public License
+/* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+:local(.tags) {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  right: 0.25rem;
+  top: 0;
+}
+
+:local(.tag) {
+  font-size: 0.8rem;
```

Comment:
Would go for `0.75em` since it is more-or-less consistent with elsewhere where we use smaller fonts. (And should actually be a variable so we can stop guessing everywhere - with the BasicToken stuff have used PostCSS variables, it does what it says on the tin. See my Less comment at the bottom)